tbx/core/models.py

```diff
@@ -199,6 +200,10 @@ class StandardPage(SocialFields, Page):
         MultiFieldPanel(SocialFields.promote_panels, "Social fields"),
     ]
 
+    search_fields = Page.search_fields + [
+        index.SearchField("body"),
```

Comment:
It might be worth checking with Anna if we are planning to add search or not. I had left that part of the field spec for now.